cit3.0-web/src/components/Flyout/Flyout.js

```diff
@@ -9,10 +10,16 @@ export default function Flyout(props) {
 
   const { children, flyoutComponent: FlyoutComponent, flyoutProps } = props;
 
+  // let waterSwitchValue = false;
+
   const toggleSidebar = () => {
     setLeftOpenState(!leftOpenState);
   };
 
+  // const updateWaterSwitchValue = (value) => {
+  //   waterSwitchValue = value;
+  // };
```

Comment:
Do you still need the switch code in here?

Reply:
No, I could have probably taken this out, but it's what I'm going to work on next so I left it in.